creator-node/src/routes/files.js

```diff
@@ -319,7 +319,7 @@ const getDirCID = async (req, res) => {
 }
 
 module.exports = function (app) {
-  app.get('/track_content_status', handleResponse(async (req, res) => {
+  app.get('/task_status', handleResponse(async (req, res) => {
```

Comment:
i feel like track_content_status is more descriptive

Reply:
I wanted to generalize this route path, because if we ever use the fileprocessingqueue for sometihng else (like image upload), `/track_content_status` is not representative. I can give this a more explicit name, like `/file_processing_task_status`?

Reply:
@dmanjunath should I change the routes to what I suggested?

Reply:
why not have
track_content_status
and
image_content_status
in the future?

they could do the same thing, but the sep endpoint would make this more clear from the client persp. as to what might be happening

Reply:
+1 to this ^

Reply:
done!